src/plugins/conditionals/README.md

```diff
@@ -113,9 +113,9 @@ EOF
 echo "key = false" > `kdb file /examples/conditionals/sub`
 
 # should fail and yield an error
-kdb export /examples/conditionals ini
-#> sub/key = false
+kdb export /examples/conditionals simpleini
```

Comment:
@sanssecours said he will add a simpleini replacement for MacOS.  #701

We should not make `ini` even more complicated.